docs/data-sources/process_automation_trigger.md

```diff
@@ -0,0 +1,32 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "genesyscloud_process_automation_trigger Data Source - terraform-provider-genesyscloud"
```

Comment:
The naming convention for resources only puts underscores between names if there is a forward slash between them in the API path, so this will need to be `genesyscloud_processautomation_trigger` in all files and references.